_posts/2024-04-26-modules-101.markdown

```diff
@@ -0,0 +1,137 @@
+---
+layout: post
+title:  "Valkey modules 101"
+authors: 
+  - dmitrypol
+date: 2024-04-26 01:01:01 -0700
+categories: update
```

Comment:
What are our categories?

Reply:
good question.  I updated the category to `modules` but I can omit it.  It becomes part of the URL so we don't want to change it afterwards or it will direct links